image/templates/helm/stackrox-central/templates/02-config-controller-00-serviceaccount.yaml

```diff
@@ -8,10 +8,10 @@ metadata:
     {{- include "srox.labels" (list . "serviceaccount" "config-controller") | nindent 4 }}
   annotations:
     {{- include "srox.annotations" (list . "serviceaccount" "config-controller") | nindent 4 }}
-      {{- if and (eq ._rox.env.openshift 4) (not ._rox.env.managedServices) }}
-      serviceaccounts.openshift.io/oauth-redirecturi.main: "sso/providers/openshift/callback"
-      {{- end }}
+    {{- if and (eq ._rox.env.openshift 4) (not ._rox.env.managedServices) }}
+    serviceaccounts.openshift.io/oauth-redirecturi.main: "sso/providers/openshift/callback"
```

Comment:
I'm not sure exactly what this does, but I assume it's for users logging in to Central.  Do you think we need this for the config-controller?

Reply:
I'm not sure either, I just followed the central pattern. Can remove.